src/EditorFeatures/CSharpTest/Classification/SemanticClassifierTests.cs

```diff
@@ -2714,7 +2714,7 @@ public async Task TestCreateWithBufferNotInWorkspace()
                 visibilityTracker: null,
                 listenerProvider);
 
-            using var tagger = (IDisposable?)provider.CreateTagger<IClassificationTag>(disposableView.TextView, extraBuffer);
+            using var tagger = provider.CreateTagger(disposableView.TextView, extraBuffer);
```

Comment:
made things more strongly typed, which had the added benefit of removing hte need for speculative casting in tests.